pages/agent/v3/cli_redactor.md

```diff
@@ -0,0 +1,9 @@
+# `buildkite-agent redactor`
+
+The Buildkite Agent automatically redacts some sensitive information from logs, such as secrets fetched with the [`secret get`](/docs/agent/v3/cli-secret) command, and any environment variables that match the value given in the [`--redacted-vars` flag](/docs/agent/v3/cli-start#redacted-vars).
+
+However, sometimes a job will source something sensitive through a side channel - perhaps a third-party secrets storage system like Hashicorp Vault or AWS Secrets Manager. In these cases, you can use the `redactor add` command to add the sensitive information to the redactor, ensuring it is redacted from subsequent logs.
```

Comment:
One last thing, we're adopting Google's developer documentation guidance on using [em dashes to represent pauses in text like this (deliberately without spacing)](https://developers.google.com/style/dashes):

```suggestion
However, sometimes a job will source something sensitive through a side channel—perhaps a third-party secrets storage system like Hashicorp Vault or AWS Secrets Manager. In these cases, you can use the `redactor add` command to add the sensitive information to the redactor, ensuring it is redacted from subsequent logs.
```

Would be nice to commit this before merging :-)

Reply:
ah shit, just barely too slow 😅

Reply:
Whoops, too late—my bad lol (another time)